tests/unit/dataactvalidator/test_fsrs.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import Mock
+
+from dataactvalidator import fsrs
+
+
+def newClient_call_args(monkeypatch, **config):
+    """Set up and create a newClient request with the provided config. Return
+    the arguments which were sent to the suds client"""
+    mock_client = Mock()
+    monkeypatch.setattr(fsrs, 'Client', mock_client)
+    config = {'fsrs_service': config}
+    monkeypatch.setattr(fsrs, 'CONFIG_BROKER', config)
+    fsrs.newClient()
+    return mock_client.call_args[1]
```

Comment:
Sure thing -- I'll update it to something like the following if it's cleared?

```python
args, kwargs = mock_client.call_args 
```

We only care about the `kwargs`, but I haven't found a good way to retrieve that by name from a `unittest.mock.call` other than decomposing the tuple.